app/javascript/pages/admin/Events/index.js

```diff
@@ -39,29 +39,35 @@ const columns = deleteEvent => [
     filterable: true,
   },
   {
-    Header: 'Office',
-    accessor: 'office.name',
-    minWidth: 70,
+    Header: 'Organization',
+    accessor: 'organization.name',
     sortable: true,
     filterable: true,
   },
   {
-    Header: 'Description',
-    accessor: 'description',
+    Header: 'Start(Date)',
+    accessor: 'startsAt',
+    width: 130,
     sortable: true,
-    filterable: true,
+    Cell: ({ value }) => moment(value).format('MMM DD, Y'),
   },
   {
-    Header: 'Start',
-    accessor: 'startsAt',
+    id: 'duration',
+    Header: 'Duration',
+    width: 80,
+    accessor: e => {
+      let start = moment(e.startsAt)
+      let end = moment(e.endsAt)
+      let diff = end.diff(start)
+      return moment.utc(diff).format('H:mm')
+    },
     sortable: true,
-    Cell: ({ value }) => moment(value).format('h:mm a, MMM D, Y'),
   },
   {
-    Header: 'End',
-    accessor: 'endsAt',
+    Header: '# Participants',
```

Comment:
`Participants` should also be fine also, since they display numbers, it's implicit it's the quantity